pkg/workload/tpcc/tpcc.go

```diff
@@ -137,6 +141,61 @@ func (w *waitSetter) String() string {
 	}
 }
 
+var _ pgx.QueryTracer = fileLoggerQueryTracer{}
+
+type fileLoggerQueryTracer struct {
+	filePath string
+	file     *os.File
+}
+
+func newFileLoggerQueryTracer(filePath string) (*fileLoggerQueryTracer, error) {
+	tracer := &fileLoggerQueryTracer{filePath: filePath}
+	file, err := os.OpenFile(tracer.filePath, os.O_CREATE|os.O_TRUNC|os.O_RDWR, 0666)
+	if err != nil {
+		return nil, err
+	}
+	tracer.file = file
+	return tracer, nil
+}
+
+func (t *fileLoggerQueryTracer) Close() {
+	if t.file != nil {
+		t.file.Close()
```

Comment:
We might want to `Sync()` the file on close too.

Reply:
Updated this -- I also updated `Close` to return an error if either `Sync` or `Close` return one. Is there a more idiomatic way to do this than the two if statements?